keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -169,6 +192,149 @@ import (
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes, the feature can be disabled once it is enabled. If disabled, users must ensure
+  any out-of-tree counterpart components are no longer running in the cluster.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  All capabilities from in-tree cloud providers will be re-enabled.
+
+* **Are there any tests for feature enablement/disablement?**
+  Adequate unit tests, component integration test and e2e tests will be added for this feature before
+  it is goes beta and on by default.
+
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+
+  TBD for beta.
+
+* **What specific metrics should inform a rollback?**
+
+   TBD for beta.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+
+  TBD for beta.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
+fields of API types, flags, etc.?**
+
+  TBD for beta.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+
+  TBD for beta.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine
+the health of the service?**
+
+  TBD for beta.
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+
+  TBD for beta.
+
+* **Are there any missing metrics that would be useful to have to improve observability
+of this feature?**
+
+  TBD for beta.
+
+### Dependencies
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **Does this feature depend on any specific services running in the cluster?**
+
+  TBD for beta.
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+
+  No, if anything it will result in reduced API calls in core components.
+
+* **Will enabling / using this feature result in introducing new API types?**
+
+  No.
+
+* **Will enabling / using this feature result in any new calls to the cloud
+provider?**
+
+  No, it will actually remove calls to the cloud provider in all core components.
+
+* **Will enabling / using this feature result in increasing size or count of
+the existing API objects?**
+
+  No, it will actually significantly reduce the size of components once in-tree
```

Comment:
It's not about components, but rather about API object.

So this should be unrelated.

Reply:
ack